job-service-db/src/main/resources/db/migration/functions/public/R__getDependentJobs.sql

```diff
@@ -20,6 +20,8 @@
  *  Description:
  *  Returns a list of dependent jobs that are now eligible to run.
  */
+DROP FUNCTION IF EXISTS get_dependent_jobs();
```

Comment:
- This should be in the VX__ scripts.
- The other functions whose parameters have changed should also be there.